.travis.yml

```diff
@@ -2,7 +2,6 @@
 language: ruby
 
 rvm:
-  - 1.9.3
```

Comment:
This should not be dropped here. If it is to be dropped, it should be done separately along with changes to the README.